tests/framework/extensions/node_status.go

```diff
@@ -0,0 +1,51 @@
+package ec2
```

Comment:
It still has ec2 as the package name, now the file is in the extensions package, but not in any subdirectory. It should be in extensions/node

Reply:
Just to be clear, you're saying to make a new directory `node` and reference the `node` package? Or just put it in the existing `nodes` folder and reference that? Just want to make clear

Reply:
We already have a nodes extension put it in there but not in ec2. And make sure to change the package name on line 1

Reply:
Got it, thanks!